config.yaml.example

```diff
@@ -16,6 +16,14 @@ integrations:
     config:
       AUTH_TOKEN: example-token
       ORGANIZATION_SLUG: example-organization
+storage:
```

Comment:
I know this is just an example file, but we technically can't have two `storage` entries in a valid config file. I'd recommend creating a separate example file for showing a Memgraph configuration.

Reply:
I created another `config.yaml.memgraph.example`, the name might be too long and I can change it if necessary